content/en/blog/getting-started/index.md

```diff
@@ -0,0 +1,252 @@
+the bot handles the rest of the issuing process (**6-7**).
+
+Once the issuer bot notices that credential issuing succeeded, it stops the bot (**10-11**), and
+the sample script moves on to verifying the same credential.
+
+```mermaid
+sequenceDiagram
+  autonumber
+    participant CLI
+    participant Issue Bot
+    participant Agency
+    participant Web Wallet
+
+    CLI->>Agency: Create invitation
+    Agency-->>CLI: Invitation URL
+    CLI-->>Web Wallet: <<show QR code>
+    CLI->>Issue Bot: Start
+    Web Wallet->>Agency: Read QR code
+    Agency-->>Issue Bot: Connection ready!
+    Issue Bot->>Agency: Issue credential
+    Agency-->>Web Wallet: Accept offer?
+    Web Wallet->>Agency: Accept
+    Agency-->>Issue Bot: Issue ready!
+    Issue Bot->>Issue Bot: Terminate
+```
+
+### CLI Script Verifying Credential
+
+Steps **1-6** proceed similarly to the issuing: first, the agents form a new pairwise connection.
+However, the process continues with a proof request sent by the verifier bot (**7**). The proof request
+contains the attributes the bot wishes the holder to reveal. The web wallet user sees
+the requested data once they receive the message (**8**), and they can either accept or reject the request.
+
+Once the proof is accepted (**9**), the agency verifies it cryptographically.
+If the verification succeeds, the agency notifies the verifier bot with the proof values (**10**).
+It can reject the proof if the values are not acceptable by the business logic.
+The sample bot accepts all attribute values, so the verifying process is continued
+without extra validation (**11**).
+The bot exits when the proof is completed (**12-13**).
+
+And that’s all core features of verified data flow executed utilizing the CLI tool!
+
+```mermaid
+sequenceDiagram
+  autonumber
+    participant CLI
+    participant Verify Bot
+    participant Agency
+    participant Web Wallet
+
+    CLI->>Agency: Create invitation
+    Agency-->>CLI: Invitation URL
+    CLI-->>Web Wallet: <<show QR code>
+    CLI->>Verify Bot: Start
+    Web Wallet->>Agency: Read QR code
+    Agency-->>Verify Bot: Connection ready!
+    Verify Bot->>Agency: Proof request
+    Agency-->>Web Wallet: Accept request?
+    Web Wallet->>Agency: Accept
+    Agency-->>Verify Bot: Proof paused
+    Verify Bot->>Agency: Resume proof
+    Agency-->>Verify Bot: Proof ready!
+    Verify Bot->>Verify Bot: Terminate
+```
+
+### CLI as a Test Tool
+
+Note that you can also utilize the CLI for testing.
+It is an excellent tool to simulate the functionality on the other end.
+
+For instance, let’s say you are developing an issuer service.
+You can use the CLI tool to manage the holder client and to receive the credential.
+Or you can use the web wallet to hold the credential
+and create another client with the CLI tool to verify the issued data.
+
+## Run the Server Samples
+
+Ok, so you have now taken over the CLI tool, but how about the integration into your application?
```

Comment:
Here we could start a new blog post that would also bring e.g. Go code blocks to the post to explain some key parts of our gRPC API. 

.. I would hate this comment as an author, more things to do. Writing and drawing sequence diagrams, ...

Still I think it's so close :-D